krkn/scenario_plugins/zone_outage/zone_outage_scenario_plugin.py

```diff
@@ -58,15 +60,22 @@ def run(
                         "Network association ids associated with "
                         "the subnet %s: %s" % (subnet_id, network_association_ids)
                     )
-                    acl_id = cloud_object.create_default_network_acl(vpc_id)
+                    
+                    # Use provided default ACL if available, otherwise create a new one
+                    if default_acl_id:
+                        acl_id = default_acl_id
+                        # Don't add to acl_id since we didn't create it
```

Comment:
Cant we specify in the comment that we aren't adding it to the **acl_ids_created** specifically because we don't want to delete it at the end of the scenario

Reply:
+1 we might want to log the message ( log.info ) for the users to understand that the provided acl is being used and it won't be deleted at the end of the scenario. Thoughts?

Reply:
Good catch, @paigerube14.

@chaitanyaenr, I'll add the log as well.